src/screens/SignupScreen.tsx

```diff
@@ -203,13 +205,14 @@ export const SignupScreen: React.FC<SignupProps> = () => {
 
 const styles = StyleSheet.create({
     scrollviewContainer: {
-        paddingTop: StatusBar.currentHeight
+        paddingTop: StatusBar.currentHeight,
+        flex: 1
```

Comment:
this flex 1 does not allow the user to scroll when filling out the form , making it hard for the user to see what they are filling in

Reply:
Oh okay, I was not aware. Great catch!